Makefile

```diff
@@ -95,13 +95,14 @@ runconformance: runservertests runclienttests
 
 .PHONY: runservertests
 runservertests: $(BIN)/connectconformance $(BIN)/referenceserver $(BIN)/grpcserver
-	$(BIN)/connectconformance --conf ./testdata/reference-impls-config.yaml --mode server $(BIN)/referenceserver
-	$(BIN)/connectconformance --conf ./testdata/grpc-impls-config.yaml --mode server $(BIN)/grpcserver
+	$(BIN)/connectconformance -v --conf ./testdata/reference-impls-config.yaml --mode server -- $(BIN)/referenceserver
+	$(BIN)/connectconformance -v --conf ./testdata/grpc-impls-config.yaml --mode server -- $(BIN)/grpcserver
+	$(BIN)/connectconformance -v --conf ./testdata/grpc-web-server-impl-config.yaml --mode server -- $(BIN)/grpcserver
```

Comment:
Since the client and server have different capabilities, I split the config into two. One represents what the client supports, which the server also supports (`grpc-impls-config.yaml`), and this new file (`grpc-web-server-impl-config.yaml`) is just the incremental cases on top of that now that the server supports the gRPC-Web protocol.